test/service/test_request_validation.py

```diff
@@ -227,9 +227,9 @@ def test_bdbag_manifest(self):
                 samples = iter(csv.reader(fh, delimiter='\t'))
```

Comment:
And `rows` instead of `samples` for this, too. They aren't actually samples. The rows represent files.

Reply:
Good point.